packages/ui/src/components/textarea.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react'
```

Comment:
Since we’ve migrated this component from `canary`, let’s try to remove it from `canary` right away by updating any imports that reference canary to the new path where the component now resides in `packages/ui`

Reply:
sure

Reply:
Once you update the paths, please don’t forget to remove this component from `canary`. In doing so, I believe there might be issues in `/packages/playground/`, so you may need to update the paths there as well for now. However, as far as I know, @knagurski  is already working on fully eliminating playground, so I think it will be easier for us to part with these components in the future. :slightly_smiling_face:

Reply:
yeah, it might pollute this PR a bit though, will need to remove references from packages/playground and packages/views as well